clang/include/clang/Driver/Options.td

```diff
@@ -3426,6 +3426,29 @@ def fpass_plugin_EQ : Joined<["-"], "fpass-plugin=">,
   MetaVarName<"<dsopath>">,
   HelpText<"Load pass plugin from a dynamic shared object file (only with new pass manager).">,
   MarshallingInfoStringVector<CodeGenOpts<"PassPlugins">>;
+def mtocdata : Flag<["-"], "mtocdata">,
+  Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>,
```

Comment:
cl/dxc don't use Power ISA. Remove them.

Use `BoolOption<"m", "tocdata", ...`

Reply:
do we need `CLOption, DXCOption` , these two option is for Window OS.